.github/workflows/deploy.yaml

```
@@ -837,7 +839,7 @@ jobs:
           slack-message-id: ${{ needs.init.outputs.slack-message-id }}
 
   deploy_ocpp_gateway:
-    if: ${{ ! failure() && ! cancelled() && ( inputs.service-identifier == 'ocpp-gateway' || inputs.service-identifier == 'ocpp-v201-gateway' ) }}
+    if: ${{ ( inputs.service-identifier == 'ocpp-gateway' || inputs.service-identifier == 'ocpp-v201-gateway' ) }}
```

The 201 specific gateways (ocpp-v201-gateway) was an attempt that was never used

```suggestion
    if: ${{ inputs.service-identifier == 'ocpp-gateway' }}
```